### DISCUSSION TITLE
Can CompreFace be used with the Apache Web server instead of with Ngnix?

### DISCUSSION THREAD

**Discussion by simdol2** [exadel-inc/CompreFace#701] [Q&A] [answered]
Can CompreFace be used with the Apache Web server instead of with Ngnix?

**Comment by pospielov** [comment, marked as answer]
I believe yes, but you will need to create Apache config by yourself. 
There are two things to solve:
1. You need to write proxy to other compreface containers as we did in nginx: https://github.com/exadel-inc/CompreFace/blob/master/ui/nginx/conf.d/nginx.conf
2. You need to serve UI using Apache, here we build it and copy to nginx folder: https://github.com/exadel-inc/CompreFace/blob/master/ui/docker-prod/Dockerfile
